charts/ingress-nginx/values.yaml

```diff
@@ -998,6 +997,26 @@ defaultBackend:
   #    effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
 
   affinity: {}
+  # -- Topology spread constraints rely on node labels to identify the topology domain(s) that each Node is in.
+  # Ref.: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
+  #  
```

Comment:
You can safely omit this line. The way we are currently doing it for `controller.topologySpreadConstraints` isn't perfect, so don't relate to that.